packages/core/package.json

```
@@ -33,6 +33,7 @@
     "cross-fetch": "^4.0.0",
     "csv-parse": "^5.5.6",
     "csv-stringify": "^6.5.1",
+    "diff": "^7.0.0",
```

this is atest comment
```suggestion
    "diff": "^7.0.x",
```